core/trino-main/src/main/java/io/trino/sql/analyzer/Analysis.java

```diff
@@ -1979,6 +2000,16 @@ public boolean isDirectlyReferenced()
             return directlyReferenced;
         }
 
+        public List<ViewLikeInfo> getReferenceChain()
+        {
+            return referenceChain;
+        }
+
+        public Optional<String> getViewText()
+        {
+            return viewText;
+        }
```

Comment:
nit: Can we swap the method arrgangement so as to be in sync with ctor argument ?